test/plugins/features/movies/controller.test.js

```diff
@@ -90,5 +108,24 @@ describe('movie controller', () => {
 
   });
 
+  describe('addLocation', () => {
+
+    it('location to movie', () => {
+      return Bluebird.all([
+        new Movie({ title: 'Bedazzled' }).fetch(),
+        new Location({ name: 'SoMa' }).fetch()
```

Comment:
nit: thoughts on using the constants up at top instead of hardcoding the strings?

```js
new Movie({ title: MOVIE_LIST[0].title }).fetch(),
new Location({ name: LOCATION_LIST[0].name }).fetch()
```